movies-search/src/MovieList/MovieContainer.jsx

```diff
@@ -29,7 +37,8 @@ const filters = ["release_date", "vote_count"];
 
 const mapDispatchToProps = (dispatch) => {
     return {
-        fetchData: () => dispatch(fetchAllMovies())
+        fetchData: () => dispatch(fetchAllMovies()),
+        searchMovie: (query) => dispatch(searchMovies(query))
```

Comment:
why do you need two special actions here? Both are just fetching movies based on query params. In one case the search text is just empty